float64 cable_ph
float64 cable_th
float64 cable_ph_dot
float64 cable_th_dot